core/ops_builtin_v8.rs

```diff
@@ -575,6 +575,33 @@ fn op_get_promise_details<'a>(
   }
 }
 
+#[op(v8)]
+fn op_set_promise_hooks<'a>(
```

Comment:
Make sure to add `op_set_promise_hooks::decl()` in `.ops()` declaration for the extension at the top of the file

Reply:
Thanks!